common/consts/consts.go

```diff
@@ -8,6 +8,7 @@ const (
 	CurrentNamespaceEnvVar        = "CURRENT_NS"
 	DefaultOdigosNamespace        = "odigos-system"
 	OdigosConfigurationName       = "odigos-config"
+	OdigosConfigurationFileName   = "config.yaml"
```

Comment:
calling it `config.yaml` makes it should as if it is mapped to a real file somewhere. do you think it might be confusing for people reading it in the future?

Reply:
It definitely **could** be mapped to a real file as well, but I don't think that's necessarily something that would confuse me if I saw this.

Happy to change it, but since the configmap is called "odigos-config", the only key, that would make more sense than "config.yaml" to me would be "content", which conceptually fits the "idea" of "odigos-config / data.content" better, but feels less practical, because 1. being overly generic 2. removing slightly more convenient volume mounting option.

What would you name it?